disco/voice/client.py

```diff
@@ -37,6 +40,13 @@ def __init__(self, msg, client):
 
 
 class UDPVoiceClient(LoggingClass):
+    # Struct format of the packet metadata
+    _FORMAT = '>HHII'
+    # Packets should start with b'\x80\x78' (32888 as a big endian ushort)
+    _CHECK = 32888
+    # Some packets will start with b'\x90\x78' (36984)
+    _CHECK2 = 36984
```

Comment:
Also the naming should be more clear, `RTC_MAGIC1` / `RTC_MAGIC2` or w/e is appropriate